docker/bootstrap/files/run.sh

```diff
@@ -28,11 +28,16 @@ if [ -n "$CASSANDRA_SEEDS" ]; then
     for cassandra in ${array[@]}
     do
         echo "Try to connect to $cassandra"
-        if nc -z -w5 $cassandra 8778
+        if nc -z -w5 $cassandra 8778 > /tmp/result.txt 2>&1
```

Comment:
I suppose you can just do this
`if nc -z -w5 $cassandra 8778 2>/tmp/result.txt`

I wouldn't expect the error to be coming from stdin